cmd/uplink/cmd/cp.go

```diff
@@ -118,6 +107,23 @@ func upload(ctx context.Context, src fpath.FPath, dst fpath.FPath, showProgress
 	return nil
 }
 
+func uploadStream(ctx context.Context, streams streams.Store, mutableObject storj.MutableObject, reader io.Reader) error {
+	mutableStream, err := mutableObject.CreateStream(ctx)
+	if err != nil {
+		return err
+	}
+
+	upload := stream.NewUpload(ctx, mutableStream, streams)
+	defer utils.LogClose(upload)
```

Comment:
Let's combine the error rather than logging it here.

Reply:
Do you mean that instead of using `utils.LogClose`, we should have an alternative `utils.CloseWithCombineError`?

Reply:
I mean using `return utils.CombineError(err, upload.Close())`

Reply:
One of the benefits of using `utils.LogClose` is that it would avoid generating an error if the handler is already closed. I think we should keep this.

Reply:
Yes, but the handler shouldn't be closed at that point. I mean the `upload.Close` should be safe to call. If there is a failure in uploading or closing then it should return an error.

Reply:
OK, you are right. https://github.com/storj/storj/pull/811